pengwin-setup.d/common.sh

```diff
@@ -52,6 +52,21 @@ echo "Removing unnecessary packages."
 sudo apt-get autoremove -y
 }
 
+function command_check() {
+# Usage: command_check <EXPECTED PATH> <ARGS (if any)>
+local execname=$(echo "$1" | sed -e "s|^.*\/||g")
+if "$execname" "$2" > /dev/null 2>&1 ; then
+	echo "Executable $execname in PATH"
```

Comment:
Please change tabs by 2 spaces

Reply:
should be resolved now, will make another PR once this one has gone through fixing up the rest of the spacing inconsistencies throughout the file with other functions